unlock-app/src/hooks/useDataForGuild.ts

```diff
@@ -25,7 +25,14 @@ export function useDataForGuild({
   network,
   recipients,
 }: UseDataForGuildProps) {
-  return useQuery(['getLockSettings', lockAddress, network], async () => {
-    return getDataForGuild(network, lockAddress, recipients)
-  })
+  return useQuery(
+    ['getLockSettings', lockAddress, network],
+    async () => {
+      return getDataForGuild(network, lockAddress, recipients)
+    },
+    {
+      refetchIntervalInBackground: true,
+      refetchInterval: 3000, // Every 3 seconds?
```

Comment:
It's excessive. What's the use case? If it is to refresh based on changing tabs or something similar, use a more specific event like refreshOnFocus or similar.

Reply:
> It's excessive. What's the use case? If it is to refresh based on changing tabs or something similar, use a more specific event like refreshOnFocus or similar.

I will look at `refreshOnFocus`!

Reply:
ho it looks like this is the default!